### PR TITLE
Migrate getting started pages

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
Copy across all content, styles and components required for the getting started pages

### CODE REVIEW

File: mosaic-site/src/components/getting-started/develop-fonts-accordion/DevelopFontsAccordion.module.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.accordionWrapper table thead {
+  position: inherit;
+}
+
+.accordionWrapper :global(.saltIcon) {
+  --salt-size-icon-base: var(--salt-size-unit);
+}
+
+.accordionWrapper :global(.saltAccordionSummary-icon) {
+  right: 0;
```
Comment (Fercas123) [inline mosaic-site/src/components/getting-started/develop-fonts-accordion/DevelopFontsAccordion.module.css:10]: I think you can do without this given it is `position:static`

**Review by Fercas123 — COMMENTED** [review]
A tiny CSS one, apart from that it all looks good to me

**Review by Fercas123 — APPROVED** [review]

File: mosaic-site/src/components/mdx/code/Code.module.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.code {
+  --code-padding-vertical: 0.1rem;
+  --code-padding-horizontal: 0.1rem;
+}
+
+.code {
+  font-family: SFMono-Regular, Menlo;
```
Comment (libertymayc) [inline mosaic-site/src/components/mdx/code/Code.module.css:7]: did we want to use --salt-typography-fontFamily-code ?

File: mosaic-site/src/css/global/code.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+pre {
+  --pre-background-color: #f6f8fa;
+  --pre-border-radius: 0.4rem;
+  --pre-color: #393a34;
+  --code-font-size: 90%;
+  --pre-line-height: 1.45;
+  --font-family-monospace: SFMono-Regular, Menlo;
+  --pre-shadow: 0 1px 2px 0 #0000001a;
+}
```
Comment (libertymayc) [inline mosaic-site/src/css/global/code.css:15]: may want to look and redefine these in future

File: mosaic-site/src/components/mdx/code/Code.module.css (hunk 2 above)
Comment (JoanaMMoreira) [reply mosaic-site/src/components/mdx/code/Code.module.css:7]: This is just to match what we currently have on the site which unfortunately comes from Docusaurus and looks slightly different than our `--salt-typography-fontFamily-code` token

File: mosaic-site/src/css/global/code.css (hunk 3 above)
Comment (JoanaMMoreira) [reply mosaic-site/src/css/global/code.css:15]: Agreed! As I mentioned above we're just trying to match our current styling for now but we should definitely revisit

**Review by libertymayc — APPROVED** [review]

**Pull request merged**